### ISSUE TITLE
Prevent comments from loading with the page

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#3891]
### Is your request related to a problem? Please describe.

It would be nice to have a way to prevent comments from loading with the page.

### Describe the solution you'd like

A new add-on that hides comments on projects and profiles until a load comments button is pressed. Ideally this add-on would prevent comments from loading altogether, but simply hiding them with `display: none` could work too.

### Describe alternatives you've considered

Ignoring the comments section or running projects in TurboWarp.

### Additional context

N/A

**Comment by TheColaber** [comment]
Related #3566... Except that one doesn't load the project.

**Comment by WorldLanguages** [comment]
What's the motivation for this?

**Comment by Samq64** [comment]
2020

**Comment by WorldLanguages** [comment]
> Apparently Scratch actually did something like this on profile pages at some point in 2020

Yeah, for server performance reasons, they're not a problem anymore

**Comment by Secret-chest** [comment]
Why close?

**Comment by Samq64** [comment]
> Why close?

Someone on the dev Discord was saying there were too many open feature requests so I closed some but I can re-open anything someone is interested in.

**Comment by BroJac5246** [comment]
> > Why close?
> 
> Someone on the dev Discord was saying there were too many open feature requests so I closed some but I can re-open anything someone is interested in.

If it's a valid suggestion it might as well stay open... it's not doing anyone any harm.
